packages/ec-core/src/operator/selector/worst.rs

```diff
@@ -0,0 +1,52 @@
+use anyhow::{Context, Result};
+use rand::rngs::ThreadRng;
+
+use super::Selector;
+use crate::population::Population;
+
+pub struct Worst;
+
+impl<P> Selector<P> for Worst
+where
+    P: Population,
+    for<'pop> &'pop P: IntoIterator<Item = &'pop P::Individual>,
+    P::Individual: Ord,
+{
+    fn select<'pop>(&self, population: &'pop P, _: &mut ThreadRng) -> Result<&'pop P::Individual> {
+        population
+            .into_iter()
+            .min()
+            .context("The population was empty")
+    }
+}
+
+#[cfg(test)]
+#[rustversion::attr(before(1.81), allow(clippy::unwrap_used))]
+#[rustversion::attr(
+    since(1.81),
+    expect(
+        clippy::unwrap_used,
+        reason = "Panicking is the best way to deal with errors in unit tests"
+    )
+)]
+mod tests {
+    use test_strategy::proptest;
+
+    use super::{Selector, Worst};
+
+    #[test]
+    fn can_select_twice() {
+        let pop = vec![5, 8, 9, 6, 3, 2, 10];
```

Comment:
just use an array
```suggestion
        let pop = [5, 8, 9, 6, 3, 2, 10];
```

Reply:
This needs to remain a `Vec` because (currently) `.select()` doesn't work on arrays. Once that is fixed, we should change this.